server/api/school-invite/school-invite.controller.js

```diff
@@ -124,6 +131,52 @@ class SchoolInvite {
       return false
     }
   }
+
+  static async accept (request, response) {
+    return handleToken(request, response, {
+      join: async (props) => {
+        const organisation = await SchoolInvite.createOrganisationFromSchool(props.schoolId)
+        await SchoolInvite.linkPersonToOrganisationAsAdmin(organisation._id, request.session.me._id)
+      }
```

Comment:
@avowkind from our conversation last week - I believe this is where you'll want to add your code to create goal cards for this person

Reply:
Thanks - I'll be able to issue a pub/sub notice here and have it handled in a more general location.